VideoWeb/VideoWeb/ClientApp/package.json

```diff
@@ -53,15 +53,19 @@
         "chart.js": "^2.9.4",
         "classlist.js": "^1.1.20150312",
         "core-js": "^3.8.2",
+        "fsevents": "^2.1.3",
         "govuk-frontend": "^3.10.2",
         "guid-typescript": "^1.0.9",
         "moment": "^2.29.1",
+        "ng-bullet": "^1.0.3",
+        "ng-mocks": "^12.4.0",
         "ng2-charts": "^2.4.2",
         "ngx-clipboard": "^14.0.1",
         "ngx-device-detector": "^2.0.5",
         "ngx-moment": "^5.0.0",
         "ngx-toastr": "^13.2.0",
         "protractor": "~7.0.0",
+        "run-script-os": "^1.1.4",
```

Comment:
`run-script-os` is not required to run this app. Please move to the `devDependencies` section